Automattic/wp-calypso
# Add "Launch site" button to site preview screen

Related to p1724614480485499-slack-C029GN3KD

## Proposed Changes

### The problem

I found it too complicated to actually launch a site after creating a Dotcom account and skip the onboarding flow by going straight to the Dashboard. You need to click the Home icon to see the site preview, then click the "Visit site" button, and finally "Launch site". This PR allows the user to launch the site from the preview screen, which makes it a bit easier to find.

#### Before

<img width="813" alt="Screenshot 2024-08-25 at 16 28 23" src="https://github.com/user-attachments/assets/8e865e7b-f3f0-486e-95e2-15df16ba166d">

<img width="372" alt="Screenshot 2024-08-25 at 16 54 26" src="https://github.com/user-attachments/assets/b4a2d224-b1de-4d5b-b148-cfe446c495f3">

#### After

<img width="813" alt="Screenshot 2024-08-25 at 16 28 27" src="https://github.com/user-attachments/assets/dbdbff75-6762-4a97-ac59-109bd4c1d5e7">

<img width="372" alt="Screenshot 2024-08-25 at 16 54 30" src="https://github.com/user-attachments/assets/9c035273-9be1-47df-9e51-80b0080ee7da">
<br><br>

It doesn't introduce any change for already launched sites.

## Why are these changes being made?

To help people with launching their sites by making the "Launch site" button easier to find.

## Testing Instructions

1. Create a Dotcom account
2. Choose a domain
3. Skip the onboarding flow by clicking the "Skip to dashboard" link
4. Click the area with "Coming Soon" button in top-left corner
5. Click the "Launch site" button and make sure it works as expected

## Pre-merge Checklist
